lib/services/history/item_history_entry.ts

```diff
@@ -14,8 +18,9 @@ export class ItemHistoryEntry {
   protected defaultContentKeyToDiffOn = 'text'
   protected textCharDiffLength = 0
   protected hasPreviousEntry = false
+  protected source: ItemHistorySource
```

Comment:
Should be marked as `readonly` as well if we don't plan on changing this value after the constructor.